### PR TITLE
Fix commitlint

### PR DESCRIPTION
**Pull request by subhod-i**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "build:dev": "webpack --mode=development && npm run build:es",
     "docs:examples": "node tooling/docs/examples-to-md.js examples/node/*.js",
     "docs:api": "node apidoc.js",
-    "lint": "commitlint --from HEAD~50 && eslint .",
+    "docs:changelog": "cp CHANGELOG.md docs/CHANGELOG.md",
```
Comment (marc0olo) [inline package.json:18]: why this again? I moved the changelog file in the release PR for reasons of linking to correct sources in repo and mkdocs documentation

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -17,6 +17,7 @@ jobs:
           key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
 
       - run: npm ci --ignore-scripts
+      - run: npx run commitlint --from HEAD~${{ github.event.pull_request.commits }} --to HEAD
```
Comment (marc0olo) [inline .github/workflows/main.yml:20]: this runs now on CI for almost 1 hour @subhod-i

**Comment by subhod-i** [comment]
@marc0olo please squash & merge.

File: package.json (hunk 1 above)
Comment (subhod-i) [reply package.json:18]: was checking it.
wait undoing this.

File: .github/workflows/main.yml (hunk 2 above)
Comment (subhod-i) [reply .github/workflows/main.yml]: Fixed it.

**Pull request merged**